.github/workflows/ccpp.yml

```diff
@@ -76,6 +90,20 @@ jobs:
       run: ninja -C build
     - name: run tests
       run: build/rx-ranges-test && build/rx-benchmark
+  MSVC-Benchmark:
+    runs-on: windows-latest
+    steps:
+    - uses: actions/checkout@v1
+    - name: install dependencies
+      run: '& "$Env:VCPKG_INSTALLATION_ROOT/vcpkg.exe" install doctest:x64-windows benchmark:x64-windows boost-date-time:x64-windows'
+    - name: make build dir
+      run: md build
+    - name: configure
+      run: 'cd build; cmake .. -DCMAKE_TOOLCHAIN_FILE="$Env:VCPKG_INSTALLATION_ROOT/scripts/buildsystems/vcpkg.cmake" -DVCPKG_TARGET_TRIPLET=x64-windows -G "Visual Studio 16 2019" -DCMAKE_PLATFORM_ARCH=x64 -DRX_TEST=ON; cd ..'
+    - name: build
+      run: cd build; cmake --build . --config Release; cd ..
+    - name: run tests
+      run: build/Release/rx-ranges-test.exe; build/Release/rx-benchmark.exe
```

Comment:
I don't now the power shell syntax (or is it cmd?) too well, but are you sure the exit code of rx-ranges-test.exe is examined in here? Maybe it simplest solution would be to run the tests in different subsections.

Reply:
Yes, good point!